codalab/model/bundle_model.py

```diff
@@ -1346,7 +1350,7 @@ def add_worksheet_items(self, worksheet_uuid, items, after_sort_key=None, replac
                 }
                 for idx, (bundle_uuid, subworksheet_uuid, value, type) in enumerate(items)
             ]
-            self.do_multirow_insert(connection, cl_worksheet_item, items_2_insert)
+            self.do_multirow_insert(connection, cl_worksheet_item, items_to_insert)
```

Comment:
Worries me a bit that the delete above and this insert aren't atomic.

Reply:
I agree; probably belongs in another issue though because we don't make multiple insertions atomic anyway. I've made https://github.com/codalab/codalab-worksheets/issues/1504